src/odemis/gui/util/img.py

```diff
@@ -2228,8 +2232,27 @@ def images_to_export_data(streams, view_hfw, view_pos,
     return (list of DataArray)
     raise LookupError: if no data visible in the selected FoV
     """
+    max_res = (MAX_RES_FACTOR * CROP_RES_LIMIT) ** 2
+    mpp = math.sqrt( (view_hfw[0]*view_hfw[1]) / max_res) # Area = [meters per pixel]^2 * number_of_pixels
+
+    def img_callback(da):
+        img_received.set()
+
+    for stream_idx, projection in enumerate(streams):
+        if hasattr(projection, 'mpp'):
+            img_received = threading.Event()
+            new_proj = RGBSpatialProjection(projection.stream)
```

Comment:
This only works if projection.mpp is equal to mpp. For a smaller mpp we want to actually decrease the resolution so no crazy big files are exported isn't it?

Reply:
If the data has too many pixels, it should be scaled down by the drawing code later in the function (just as for the standard data). That said it's a little bit theoretical, because the chances that mpp is smaller than min_mpp are very slim, as I think the only way to get to there would be to have a really big screen... so maybe not worthy for an optimisation.